src/Pirouette/Symbolic/Prover.hs

```diff
@@ -186,7 +188,7 @@ worker resultVar bodyTerm assumeTerm proveTerm = do
       -- debugPrint (pretty proveTerm')
       -- debugPrint somethingWasEval
       -- check the fuel
-      noMoreFuel <- gets sestStoppingCondition >>= \s -> s <$> currentStatistics
+      noMoreFuel <- asks (stoppingCondition . seeOptions) >>= \s -> s <$> currentStatistics
```

Comment:
Probably worth taking a chance to rename the binding to smth like `shouldStop`, since it's not necessarily about the fuel now.

Also, the comment above is stale, I'd delete it because the code here is fairly straightforward.